txtorcon/torcontrolprotocol.py

```diff
@@ -102,7 +102,40 @@ def unlisten(self, cb):
         self.callbacks.remove(cb)
 
     def got_update(self, data):
-        # print self.name,"got_update:",data
+        for cb in self.callbacks:
+            cb(data)
+
+class StemEvent(Event):
+    """
+    A class which parses the event information into stem.response.ControlMessage
+    instance.
+    https://stem.torproject.org/api/response.html#stem.response.ControlMessage
+
+    The information is then parsed into stem.response.events.Event subclass instance
+    https://stem.torproject.org/api/response.html#stem.response.events.Event
+    """
+    def got_update(self, data):
```

Comment:
Sorry, another nit: can you turn these into proper Sphinx-syntax links? Should be an example in some other comment somewhere. Bonus points if we can "inter-Sphinx" link them, like Twisted links work (I believe?)